<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confectionery</class>
 <widget class="QDialog" name="confectionery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addProduct">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeProduct">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToOrder">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить в заказ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="myOrder">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Мой заказ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="historyOfOrders">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>История заказов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
